### PR TITLE
Fix bug that showed error in logs for mapping visas when there is no …

### LINKED ISSUE #54: Handeling empty visas when they are not needed
In a case like uploading when an authorized passport is sufficient to proceed, if the visa list is empty an error appears in the log. It does not stop the execution but it can cause confusion and is not ideal.

### PR DESCRIPTION
**Pull request by Parisa68**
…visa.

Fixes #54

### Describe the pull request:

- [x] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Unit/integration tests
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change

### CODE REVIEW

**Review by kjellp — COMMENTED** [review]
@Parisa68 Had a quick look, and the code seems to be doing the correct thing :) But noticed that there is a conflict flagged below in the PR thread, maybe something you need to look at before Joshua can do the final approval and merge?

**Comment by joshbaskaran** [comment]
It was a minor issue with the imports. Fixed now.

**Review by joshbaskaran — APPROVED** [review]
LGTM :)

**Pull request merged**
